main.go

```diff
@@ -398,7 +398,7 @@ func main() {
 					&http.Client{Transport: t},
 					cfg.server.healthcheckURL,
 					http.MethodGet,
-					http.StatusNotFound,
+					http.StatusOK,
```

Comment:
nit: Does it make more sense to return a 204 (no content) if it's simply for liveness?